reporters_db/data/reporters.json

```diff
@@ -16437,6 +16485,7 @@
                 "Pac.Rep.": "P.",
                 "Pacific": "P.",
                 "Pacific Rep.": "P.",
+                "Page": "P.",
```

Comment:
This looks surprising.

Reply:
Indeed, I found some citations with Page as reporter, the pdf also comes with Page reporter, see first and last page, e.g. [259 Page 821] (https://cite.case.law/nm/32/517/) [260 Page 411](https://cite.case.law/nm/32/518/)

I searched in other platforms and the Page reporter is always mapped to Pacific Reporter (P.) Even in courtlistener is mapped to P. https://www.courtlistener.com/opinion/3590876/carman-v-bd-of-comrs-of-mckinley-co/ or https://www.courtlistener.com/opinion/3591003/ford-v-norton/

Maybe those in courtlistener were fixed manually

Reply:
@mlissner - this looks right to me.  It looks like the New Mexico reports - just wrote page in as a stand in for the pacific reporter at the time.  I double checked - that there wasn't some reporter named Page.  There was one series... Page on Divorce, Page of Wills Page on Contracts, but it seems unrelated.

Reply:
Cool. Maybe a comment in the JSON, then?

Reply:
that seems right.